common/check_internal.cpp

```diff
@@ -23,6 +23,8 @@ ExitingStream::~ExitingStream() {
 
 auto ExitingStream::Done() -> void {
   buffer_ << "\n";
+  buffer_.flush();
```

Comment:
How did this come up for the change?

Reply:
A lifetime issue with the array of interface arguments caused the tests to fail with opt builds but not with dbg or fastbuild, and the opt builds also seemed to lose the end of the check failure diagnostic if I didn't explicitly flush the stream.